### ISSUE TITLE
Text over refresh button

### ISSUE THREAD

**Issue by Fixer-007** [minetest/minetest#7088]
##### Issue type

 - Bug report
 
##### Minetest version

```
minetest 0.5.0-673f7196
```

##### OS / Hardware

Operating system: Windows 7
CPU: Intel Core i3-2120

GPU model: ATI X800GTO
OpenGL version: 2.1

##### Summary
Text is displayed on top of refresh button after it was introduced via https://github.com/minetest/minetest/pull/6957 ![36508838-84c1d7a2-1766-11e8-9ed6-8014471c115e](https://user-images.githubusercontent.com/16494741/36569796-e1524e58-1837-11e8-9918-32367a1db01b.png)
Approximated text:  "(pro?)jects/min(?etest?)"

**Comment by paramat** [comment]
I can't reproduce this. Looks like part of a filepath.

**Comment by Fixer-007** [comment]
I don't have projects in filepath.

**Comment by HybridDog** [comment]
I tried to reproduce it:
![2018-03-03-205934_1920x1056_scrot](https://user-images.githubusercontent.com/3192173/36938707-e39f0026-1f25-11e8-8bbc-48fe0c7e4b1d.png)
However, it's behind the button, not on top on it.

It looks like this with big window size:
![2018-03-03-210513_1920x1080_scrot](https://user-images.githubusercontent.com/3192173/36938759-badedc1e-1f26-11e8-8e07-943083854fa4.png)

**Comment by paramat** [comment]
Thanks, what text is this?

**Comment by paramat** [comment]
I edited the image to remove an email address.

**Comment by sfan5** [comment]
The result of calling `gettext`() with an empty string:
https://github.com/minetest/minetest/blob/master/po/de/minetest.po#L8

**Comment by paramat** [comment]
Yeah done in https://github.com/minetest/minetest/commit/5ef9056e89855660d93c4e8d98a7dbc349c1a7eb

**Comment by Fixer-007** [comment]
Ha, just wanted to comment on this! I've suspected very same thing, after installing minetest without gettext support button was normal, so yeah... it is linked to gettext.

**Comment by HybridDog** [comment]
@nerzhul merged that.

**Comment by paramat** [comment]
@ThomasMonroe314

**Comment by paramat** [comment]
So what's the fix?
`fgettext(" ")`
?

**Comment by rubenwardy** [comment]
The fix is either consistently not calling gettext("") or removing the "" entry from the PO files

**Comment by sfan5** [comment]
![Bildschirmfoto - 30.10.2013 - 16:09:14](https://user-images.githubusercontent.com/1042418/37060251-fd8f177a-218f-11e8-88da-518b60eabe07.png)
I think this applies here ;-)

**Comment by HybridDog** [comment]
>So what's the fix?
fgettext(" ")
?

I think there's no translation for " " and it's invisible, so it should work. However, to kiss you could not translate it in lieu. Then you could adhere to "" because it's not longer translated.

**Comment by paramat** [comment]
Ok, ok, i was assuming the fix wasn't trivial =) If it was trivial i still think the correct fix should be specified here instead of assuming everyone knows it.

**Comment by minduser00** [comment]
Sorry for mentioning this in my fork, without checking that this patch was compatible with all systems. I didn't think anything seemingly simple would create a problem for some compilers.

**Comment by paramat** [comment]
So we just remove ` fgettext("")` from the line?

**Comment by HybridDog** [comment]
Yes, you do, an empty string shouldn't become translated.
